src/app/home/pages/TokensPage/helpers.ts

```diff
@@ -0,0 +1,37 @@
+import moment from 'moment-timezone';
+import { IToken } from '../../../token/duck/types';
+import { StatusType } from '../../../common/components/StatusIcon';
+
+export const getTokenInfo = (token: IToken) => {
+  const expirationTimestamp = token.MigToken.status.expiresAt;
+  const expirationMoment = moment(expirationTimestamp);
+  const formattedExpiration = expirationMoment
+    .tz(moment.tz.guess())
+    .format('DD MMM YYYY, hh:mm:ss A z');
+  const hoursUntilExpiration = expirationMoment.diff(moment(), 'hours', true);
+  let statusType: StatusType;
+  let statusText: string;
+  if (hoursUntilExpiration < 0) {
+    statusType = StatusType.ERROR;
+    statusText = 'Expired';
+  } else if (hoursUntilExpiration < 1) {
+    // NATODO is 1 hour the correct threshold here?
```

Comment:
Think 1 hour is a fine place to start with, can we break this out with a named const so it's easy to identify and change later?

Reply:
Done.